neoload/README.md

```diff
@@ -0,0 +1,49 @@
+# NeoLoad integration
+
+## Overview
+
+[Tricentis NeoLoad][1] simplifies and scales performance testing for APIs and microservices, as well as end-to-end application testing through protocol and browser-based capabilities.
+
+With the NeoLoad integration, you can track performance metrics of NeoLoad tests to:
+
+- Correlate application performance with load testing metrics.
+- Analyze and visualize NeoLoad metrics using the NeoLoad Datadog Dashboard or Metrics Explorer.
+
+## Setup
+
+### Configuration
+
+For detailed instructions on NeoLoad configuration, follow the [NeoLoad documentation][2]. Choose which metrics to send in the "Push Counters" configuration of the Datadog Connector in NeoLoad. Available since NeoLoad 9.1.
+
+When Datadog receives `NeoLoad.*` metrics, the NeoLoad integration tile is installed automatically, and the default NeoLoad dashboard is added to your dashboard list.
```

Comment:
Users will need to click the install button in Datadog in order to see the dashboard.

Reply:
Ok, what configuration is needed to install automatically the integration ?
We used docker integration, and MySQL and the Dashboards were added automcatically in the dashboard's list. It would be nice for us if the NeoLoad dashboard is added when Datadog receives metrics from NeoLoad. Do you know how to d othat ?

Reply:
Hey @guillaumebert - Customers will need to:
1. Go to the [integration](https://app.datadoghq.com/integrations) page in datadog
2. Click on the NeoLoad tile (once it is pushed live) 
3. Click install 

Once they do that, the dashboard will appear. There isn't a way to accomplish this without having customers click the "install" button. For further context, the install button adds the customers account and uninstall button will remove the account from the integration.

Reply:
@guillaumebert it looks like this still needs to be updated. Could you make the changes we spoke about above?